tla-io/src/test/scala/at/forsyte/apalache/tla/imp/TestSanyImporter.scala

```diff
@@ -500,7 +500,7 @@ class TestSanyImporter extends SanyImporterTestBase {
     def expectDecl(name: String, body: TlaEx): Unit =
       findAndExpectOperDecl(root, name, List(), body)
 
-    val trueOperDecl = mod.declarations(1)
+    mod.declarations(1)
     expectDecl("True", ValEx(TlaBool(true)))
```

Comment:
There is something wrong about this test most likely

Reply:
It looks to me like this was just never used (looking at the git blame history), and I think it's any how checked for at line 504, which checks that the `True` operator is defined as the RHS.